CouplingDataUser.H

```diff
@@ -1,11 +1,20 @@
 #ifndef COUPLINGDATAUSER_H
 #define COUPLINGDATAUSER_H
 
+#include "Utilities.H"
 #include <vector>
 #include <string>
 
 namespace preciceAdapter
 {
+// A small enum to deal with the different locations of the
+// coupling interface
+enum class LocationType
+{
+    none,
+    faceCenters,
+    faceNodes
+};
```

Comment:
Yes, this was also my first thought. The problem is the include hierarchy since the `CouplingDataUser` is required in the `Interface` and the `locationType` is eventually stored in the `CouplingDataUser`. If you don't like it here, we probably need an additional header or another file to put it.